auth/src/main/java/com/firebase/ui/auth/ui/email/RegisterEmailActivity.java

```diff
@@ -168,6 +334,7 @@ public void onFailure(@NonNull Exception e) {
                         } else if (e instanceof FirebaseAuthUserCollisionException) {
                             // Collision with existing user email
                             emailInput.setError(getString(R.string.error_user_collision));
+                            checkAccountExists(mEmailEditText.getText().toString());
```

Comment:
Come to think of it, this could be our error handling if `fetchProvidersForEmail` fails: the user can't sign up if a user already exists and we will retry to loading that user.